src/components/datasets/files/custom-actions-dialog/CustomActionsDialog.vue

```diff
@@ -152,27 +171,47 @@ export default {
     closeDialog: function() {
       this.visible = false
     },
+    /**
+     * Formats the data from the selected files for the integrations API.
+     */
+    formatSelectedFilesForAPI: function() {
+      console.log('selectedFiles', this.selectedFiles)
+
+      const presignedUrls = []
+
+      this.selectedFiles.forEach(file => {
+        const packageId = pathOr('', ['content', 'id'], file)
+        presignedUrls.push({
+          filename: file.content.name,
+          url: `${this.config.apiUrl}/packages/${packageId}/files/${
+            file.content.id
+          }?short=true&api_key=${this.userToken}`
+        })
+      })
+      return presignedUrls
+    },
     /**
      * Makes API Call to run custom event on target
      */
     runCustomEvent: function() {
       // const fileIds = this.selectedFiles.map(item => item.content.id)
-      const url = `https://api.pennsieve.net/datasets/N:dataset:e2eaa76a-e0e0-4f25-97a8-6fa2bf6dd83f/event`
-      const message = JSON.stringify({
-        organizationId: `${this.activeOrganization.organization.id}`,
-        datasetId: `${this.dataset.content.id}`,
-        eventCategory: 'CATEGORY',
-        eventType: 'CUSTOM'
+
+      const url = `https://api2.pennsieve.net/integrations`
```

Comment:
We should dynamically create this URL as it is different for PROD and DEV. Currently, the app would always reach out to dev.